### DISCUSSION TITLE
Better SVG support needed.

### DISCUSSION THREAD

**Discussion by taccess0** [nomacs/nomacs#1029] [General]
Mask cropped parts and hidden objects/layers are still shown. It's been like this since way back. I recently updated nomacs and the problem still exists. Even Chrome now has better Svg render. Thanks a lot!

**Comment by leejuyuu** [comment]
Hello @tomaccess0, does [this issue](https://github.com/nomacs/nomacs/issues/487) describe the problems you encountered? If that is the case, please consider leaving a comment there or giving a thumbs-up to raise its importance. Thanks!

**Reply by taccess0** [reply]
Did what you suggested, thanks
